### DISCUSSION TITLE
frozen terminal

### DISCUSSION THREAD

**Discussion by kasraTM** [community/community#39631] [Codespaces]
### Select Topic Area

Bug

### Body

hey,

I am using the remote VS code for cs50, and my terminal is blocked(frozen) no idea how to fix it, even the color is kinda grey'ish
<img width="1681" alt="Screen Shot 2022-11-21 at 2 18 54 AM" src="https://user-images.githubusercontent.com/114722176/202988512-61bb2670-49ff-46cd-bd47-27b7072c4e60.png">

**Comment by ossanna16** [comment]
Hi there @kasraTM and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by alexbjorlig** [comment]
I often get this issue as well... Simply stuck in: 

<img width="1737" alt="Screenshot 2023-02-21 at 19 10 24" src="https://user-images.githubusercontent.com/9967422/220425800-b6e46e50-5c11-45d2-a6dc-6e1bf7536fbb.png">

**Reply by kasraTM** [reply]
i fixed mine do you still need help ?

**Reply by alexbjorlig** [reply]
What did you do to fix it? It happens to me often, primarily after rebuilding a container.

**Reply by kasraTM** [reply]
go to setting  --> search terminal  -->  look for this """"Terminal › Integrated: Gpu Acceleration""""   put it on off or auto,  most of the times "off" should work,   on my windows system it is on auto and it works but on my mac and ipad it is off 


lemme know what happens
